client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -1,33 +1,74 @@
 /**
  * @jest-environment jsdom
  */
-import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { EditGravatar } from 'calypso/blocks/edit-gravatar';
 import ImageEditor from 'calypso/blocks/image-editor';
-import DropZone from 'calypso/components/drop-zone';
-import VerifyEmailDialog from 'calypso/components/email-verification/email-verification-dialog';
 import FilePicker from 'calypso/components/file-picker';
-import Gravatar from 'calypso/components/gravatar';
 import { AspectRatios } from 'calypso/state/editor/image-editor/constants';
-
-jest.mock( 'event', () => require( 'component-event' ), { virtual: true } );
-jest.mock( 'calypso/lib/oauth-token', () => ( {
-	getToken: () => 'bearerToken',
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+
+jest.mock( 'calypso/components/email-verification/email-verification-dialog', () => () => (
+	<div data-testid="verify-email-dialog" />
+) );
+jest.mock( 'calypso/components/drop-zone', () => ( { children } ) => (
+	<div data-testid="dropzone">{ children }</div>
+) );
+jest.mock( 'calypso/blocks/image-editor', () =>
+	jest.fn( ( { onDone } ) => (
+		<div data-testid="image-editor">
+			<button data-testid="image-editor-done" onClick={ () => onDone() }>
+				Done
+			</button>
+		</div>
+	) )
+);
+jest.mock( 'calypso/components/gravatar', () => () => <div data-testid="gravatar" /> );
```

Comment:
Instead of mocking `Gravatar`, could we just pass an `alt` or `title` attribute and grab the `img` by using `getByAltText` or `getByTitle`?

Reply:
Did that with https://github.com/Automattic/wp-calypso/pull/66808/commits/51e9f6aceab174dba3defd213a890adc4b4caeeb. Let me know if that works for you.